app/assets/javascripts/admin/addon/templates/modal/admin-customize-form-template-view.hbs

```diff
@@ -0,0 +1,22 @@
+<DModalBody @rawTitle={{this.model.name}}>
+  <HighlightedCode @lang="yaml" @code={{this.model.template}} />
+  {{! ? TODO(@keegan): Perhaps add what places (ex. categories) the templates are active in }}
+</DModalBody>
+<div class="modal-footer">
+  <DButton
+    class="btn-primary"
+    @action={{this.editTemplate}}
+    @icon="pencil-alt"
+    @label="admin.form_templates.view_template.edit"
+  />
+  <DButton
+    class="btn-danger"
```

Comment:
Super-minor, but I would have the "Delete" button right-aligned so that it is far from the "Save" action.